tests/rules/test_KMSEnabledKeyRotationRule.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from _pytest.fixtures import fixture
```

Comment:
Unsure on this import - also on L10 there is a commented out `from pytest import fixture` which I think would work better instead 👍